src/services/cohortInfos.ts

```diff
@@ -151,11 +151,13 @@ const fetchCohort = async (cohortId: string | undefined): Promise<CohortData | u
     const cohortResult: CohortData = {}
     const response = getApiResponseResources(
       await api.get<FHIR_API_Response<IEncounter | IPatient | IGroup>>(
-        `/Patient?_has:Group:member:_id=${cohortId}&_revinclude=Group:member&_revinclude=Encounter:patient`
+        `/Patient?_has:Group:member:_id=${cohortId}&_revinclude=Group:member&_revinclude=Encounter:patient&_count=10000`
```

Comment:
_count=1000 is enough

Reply:
no ?

Reply:
I may go for a little bit more than 1000, 1500 ? It's also fetching groups

Reply:
_count only affects the resource you're searching for, not the included results

Reply:
so there will be max 500 patients for the demo